edgelet/Makefile

```diff
@@ -134,6 +135,7 @@ deb: release
 	cd $(TARGET)/$(PACKAGE) && dpkg-buildpackage $(DPKGFLAGS)
 
 rpm:
+	$(INSTALL_DATA) -D $(srcdir)/contrib/centos/00-aziot-edged.preset $(presetdir)/00-aziot-edged.preset
```

Comment:
00 will cause this to run first, right? Is that the recommended approach?

Reply:
Yes, it should be fine. This is just a default, the user can override it with systemctl.